src/modules/dashboard/components/Extensions/ExtensionDetails.tsx

```diff
@@ -360,19 +358,15 @@ const ExtensionDetails = ({
                           text={chunks}
                         />
                       ),
+                      link0: extension.descriptionLinks?.[0],
```

Comment:
This is a bit of a nit-pick but I really hate the name of the link variable :)

Reply:
I can change if you have a preference for another name? How about `externalLink`? I am happy with any other if you have any suggestions. It needs to be generic as there are several in the array...

Reply:
I know, that's why it was a nit-pick. There's not really a good way to do naming here.

Leave it like it is, but I still don't like it :)